.travis.yml

```
@@ -1,47 +1,41 @@
+# https://github.com/travis-ci/travis-ci/issues/4794#issuecomment-143758799
+python: "3.5"
 
 sudo: false
 
 env:
     - TOX_ENV=py27-flake8
-    - TOX_ENV=py26-django1.5-drf2.3
-    - TOX_ENV=py26-django1.6-drf3.1
-    - TOX_ENV=py27-django1.5-drf2.3
-    - TOX_ENV=py27-django1.5-drf2.4
-    - TOX_ENV=py27-django1.5-drf3.0
-    - TOX_ENV=py27-django1.5-drf3.1
-    - TOX_ENV=py27-django1.6-drf2.3
-    - TOX_ENV=py27-django1.6-drf2.4
-    - TOX_ENV=py27-django1.6-drf3.0
-    - TOX_ENV=py27-django1.6-drf3.1
-    - TOX_ENV=py27-django1.7-drf2.3
-    - TOX_ENV=py27-django1.7-drf2.4
-    - TOX_ENV=py27-django1.7-drf3.0
-    - TOX_ENV=py27-django1.7-drf3.1
     - TOX_ENV=py27-django1.8-drf2.3
     - TOX_ENV=py27-django1.8-drf2.4
     - TOX_ENV=py27-django1.8-drf3.0
     - TOX_ENV=py27-django1.8-drf3.1
-    - TOX_ENV=py33-django1.5-drf2.3
+    - TOX_ENV=py27-django1.8-drf3.2
+    - TOX_ENV=py27-django1.8-drf3.3
+    - TOX_ENV=py27-django1.9-drf3.3
     - TOX_ENV=py33-django1.8-drf3.1
-    - TOX_ENV=py34-django1.5-drf2.3
-    - TOX_ENV=py34-django1.5-drf2.4
-    - TOX_ENV=py34-django1.5-drf3.0
-    - TOX_ENV=py34-django1.5-drf3.1
-    - TOX_ENV=py34-django1.6-drf2.3
-    - TOX_ENV=py34-django1.6-drf2.4
-    - TOX_ENV=py34-django1.6-drf3.0
-    - TOX_ENV=py34-django1.6-drf3.1
-    - TOX_ENV=py34-django1.7-drf2.3
-    - TOX_ENV=py34-django1.7-drf2.4
-    - TOX_ENV=py34-django1.7-drf3.0
-    - TOX_ENV=py34-django1.7-drf3.1
+    - TOX_ENV=py33-django1.8-drf3.2
+    - TOX_ENV=py33-django1.8-drf3.3
     - TOX_ENV=py34-django1.8-drf2.3
     - TOX_ENV=py34-django1.8-drf2.4
     - TOX_ENV=py34-django1.8-drf3.0
     - TOX_ENV=py34-django1.8-drf3.1
+    - TOX_ENV=py34-django1.8-drf3.2
+    - TOX_ENV=py34-django1.8-drf3.3
+    - TOX_ENV=py34-django1.9-drf3.3
+    - TOX_ENV=py35-django1.8-drf3.3
+    - TOX_ENV=py35-django1.9-drf3.3
 
 matrix:
+  allow_failures:
+    - env: TOX_ENV=py27-django1.9-drf3.3
+    - env: TOX_ENV=py34-django1.9-drf3.3
+    - env: TOX_ENV=py35-django1.9-drf3.3
```

We should probably fix the tests rather than suppress the errors?